openpype/hosts/max/api/lib.py

```diff
@@ -312,3 +312,98 @@ def set_timeline(frameStart, frameEnd):
     """
     rt.animationRange = rt.interval(frameStart, frameEnd)
     return rt.animationRange
+
+
+def unique_namespace(namespace, format="%02d",
```

Comment:
There seems to be a `uniqueName` method to do this?
https://help.autodesk.com/view/MAXDEV/2022/ENU/?guid=GUID-D1D7EB56-A370-4B07-99B4-BC779FB87CAF

In the "Associated Methods" section, wouldn't be better to use that?

Reply:
Unqiue Name is more like re-naming two teapots into "Teapot001", "Teapot002" for example. It's like MergeMaxFile with Auto Rename Duplicate being enabled. It is doable for load function but not sure if it can be done in up-versioning and versioning setup as it will give the new name to the objects and not sure if the system in Max is wise enough to find the last merged nodes. (Some loaders like MaxScene loader would subtract the last merged nodes in previous import from the merged nodes in the latest import and remove the objects which doesn't exist in the latest import)